src/node_modules/queries/util.mjs

```diff
@@ -1,13 +1,14 @@
 import { promises as fs } from 'fs'
+import * as pathLib from 'path'
```

Comment:
In general I think it's preferrable to use `import path from 'path'` and never use `path` as a variable to avoid confusion (this applies to the whole Node API but to npm libraries we might use).
Then wherever we need a var to express a path I prefer `filepath`, `filePath`, `dirpath`, `dirPath`.